python/tests/test_logistic_regression.py

```diff
@@ -368,15 +369,31 @@ def test_params(tmp_path: str, caplog: LogCaptureFixture) -> None:
             {"verbose": True},
             {"verbose": True},
         ),
-    ],
-)
-def test_lr_copy(
-    input_spark_params: Dict[str, Any],
-    cuml_params_update: Optional[Dict[str, Any]],
-) -> None:
-    from .test_common_estimator import _test_est_copy
+    ]
+
+    for pair in param_list:
+        input_spark_params = pair[0]
+        cuml_params_update = pair[1]
+        _test_est_copy(LogisticRegression, input_spark_params, cuml_params_update)
+
+
+def test_lr_model_copy() -> None:
+
+    from .test_common_estimator import _test_model_copy
+    from .utils import get_toy_model
+
+    model_params: List[Dict[str, Any]] = [
+        {"featuresCol": "fea_dummy"},
```

Comment:
Are these the only params that make sense to change for a model copy?

Reply:
Yes, I think so. Other params (e.g. fit_intercept, regParam) are for fit only